src/NuGetPackageVerifier/third-party.json

```diff
@@ -17,7 +17,7 @@
 
   "packages": {
     "Microsoft.ApplicationInsights": [ "2.4.0-beta3", "2.4.0-beta5" ],
```

Comment:
I only found one instance in our repos https://github.com/aspnet/AzureIntegration/pull/75.